build.sbt

```diff
@@ -68,14 +76,18 @@ lazy val anorm = project
       // New functions
       missMeth("anorm.Sql.unsafeStatement$default$2"),
       missMeth("anorm.Sql.unsafeResultSet"),
-      missMeth("anorm.Sql.unsafeStatement")),
+      missMeth("anorm.Sql.unsafeStatement"),
+      missMeth("anorm.ToStatement.contramap"),
```

Comment:
That's not true. Most commonly the conflict occurs through transitive dependencies. e.g. your project uses version 2.5.4 but you have an external lib using 2.5.3. SBT will by default pick the newer version, making the external library unusable. Actually, there are already a number of libraries that haven't updated their anorm version to latest:
 - https://github.com/mhzajac/anorm-relational
 - https://github.com/lucidsoftware/relate
 - https://github.com/giabao/1pay

Sure, in a perfect world, everyone would update and release their library as soon as a new release of any dependency comes out, but that can be quite time consuming to do, and it's just not reasonable to expect that all the time. That's why we have binary compatibility guarantees.